### PR TITLE
Simplify URL structure and add minor formatting changes

### PR DESCRIPTION
**Pull request by MaxGhenis**

### CODE REVIEW

File: _posts/2020-11-23-demo-day-2-tax-cruncher.md [hunk 1]
```diff
@@ -4,22 +4,24 @@ layout: post
 description: How to calculate a taxpayer's liabilities under current law and under a policy reform.
 categories: [demo-days]
```
Comment (jdebacker) [inline _posts/2020-11-23-demo-day-2-tax-cruncher.md:5]: @MaxGhenis What do you think about having more tags if possible?  I think this should definitely have "individual-income-tax" and "tax-cruncher" tags, at least.  I can see a lot of use by filtering out posts that talk about modeling different types of policies or using different models.

**Comment by jdebacker** [comment]
@MaxGhenis Why does each sentence need a new line?  Seems like an odd way to have to compose a post.

File: _posts/2020-11-23-demo-day-2-tax-cruncher.md (hunk 1 above)
Comment (MaxGhenis) [reply _posts/2020-11-23-demo-day-2-tax-cruncher.md]: 👍 done

**Comment by MaxGhenis** [comment]
> @MaxGhenis Why does each sentence need a new line? Seems like an odd way to have to compose a post.

This is for reviewing. When a single code line contains multiple sentences, it can be hard for reviewers to call out suggestions or questions with respect to specific parts. Also avoids long horizontal scrolls when viewing raw. The TeX StackExchange [suggests](https://tex.stackexchange.com/a/325512/151759) limiting line length even further to 80 characters, but that seems unnecessary here since we're not debugging individual lines. If others disagree we can discuss, but I've been doing it in other Markdown documents for a few months and found it helpful.

**Comment by jdebacker** [comment]
> When a single code line contains multiple sentences, it can be hard for reviewers to call out suggestions or questions with respect to specific parts.

Got it - this makes sense.  Thanks for explaining.

**Pull request merged**